doc/reference/cmdline.rst

```diff
@@ -551,8 +555,34 @@ Scaffolding is available via the :command:`odoo-bin scaffold` subcommand.
     a template directory, files are passed through jinja2_ then copied to
     the ``destination`` directory
 
+
+This will create module *my_module* in directory */addons/*.
+
+.. _reference/cmdline/populate:
+
+Database Population
+===================
+
+.. program:: odoo-bin populate
+
+Odoo CLI supports database population features.  If the feature is implemented on a given model
```

Comment:
Double-spacing is illegal.

Reply:
https://www.theatlantic.com/science/archive/2018/05/two-spaces-after-a-period/559304/